slasher/Cargo.toml

```diff
@@ -36,6 +38,8 @@ mdbx = { package = "libmdbx", git = "https://github.com/sigp/libmdbx-rs", tag =
 lmdb-rkv = { git = "https://github.com/sigp/lmdb-rs", rev = "f33845c6469b94265319aac0ed5085597862c27e", optional = true }
 lmdb-rkv-sys = { git = "https://github.com/sigp/lmdb-rs", rev = "f33845c6469b94265319aac0ed5085597862c27e", optional = true }
 
+redb = { version = "1.2.0", optional = true }
```

Comment:
While resolving the merge from unstable I changed this to point to the release. Hopefully we can stay on upstream redb!